x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/authorization/all_rules_read_only.cy.ts

```diff
@@ -34,8 +34,8 @@ describe('All rules - read only', { tags: ['@ess', '@serverless', '@skipInServer
   });
 
   beforeEach(() => {
-    login(ROLES.reader);
-    visitRulesManagementTable(ROLES.reader);
+    login(ROLES.t1_analyst);
```

Comment:
Didn't this fix make the test pass?
Looks like we could enable it on serverless, but it is still tagged `@skipInServerless`

Reply:
Noticed that you only enabled the Cypress tests to run in Serverless for `x-pack/test/security_solution_cypress/cypress/e2e/detection_response/prebuilt_rules/prebuilt_rules_install_update_authorization.cy.ts` but didn't for all else in this PR which are currently skipped, although it looks that the user/role issue was fixed for them.

Is that just out of scope of this PR?

Reply:
My plan was to unskip one/two test suites just to make sure my PR works and address the other tests in a separate PR as it may require some additional effort.